tests/integration/security/extra_trust_anchor/main_test.go

```diff
@@ -0,0 +1,69 @@
+//  Copyright 2019 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package extratrustanchor
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/citadel"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+)
+
+var (
+	ist           istio.Instance
+	g             galley.Instance
+	p             pilot.Instance
+	remoteCitadel citadel.Instance
+)
+
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+
+	cfg.Values["global.mtls.enabled"] = "true"
+
+	// disable control auth so services with credentials signed by different roots can
+	// connect to Pilot.
+	cfg.Values["global.controlPlaneSecurityEnabled"] = "false"
```

Comment:
hmm so multi trust root ca does not support the control plane mTLS? What blocks us from getting test passing with `global.ControlPlaneSecurityenabled = true`?

Reply:
This is a limitation of how and not what we're testing. The local services and remote services are sharing the same control plane (Galley/Pilot/Mixer) with different CAs. The remote envoy cannot connect to control plane with auth enabled because of the mismatch. In practice, the remote envoy would connect via a gateway or its own local control plane.

Starting another parallel in-cluster control plane required more work in the test framework. When multi-cluster support is ready in the framework we can update this test to use more than one cluster.